### ISSUE TITLE
v4 notes

### ISSUE THREAD

**Issue by stancl** [stancl/tenancy-docs#165]
Writing down notes to mention in v4 docs, so that we can clean up the issues here.
- Mention that there will be multiple connections open (= that the central connection remains open) #162 
- Mention using `Artisan::call('tenants:run', ...)` #120 
- spatie/laravel-backup integration (not tested) #101 
- Write a guide for integrating Scout #72 
- --skip-failing on tenant migrations, #73 
- Media library integration #78 
- Mention that Horizon works on the central domain #75

**Comment by ianrobertsFF** [comment]
Overhauling the docs in general would be a good idea, there's plenty of stuff in the v3 docs that are still missing/todo. 

Understandable not updating those *now* given that v4 is coming, but ideally v4 shouldn't release with similar "see v3/v2 docs" or "TODO", etc.

**Comment by abrardev99** [comment]
Can we make headings clickable in `v4`? It's actaully just wrapping them in `a` tag. I see almost all headings have the `id` attribute so that we can use `id` in `href`. 
I think it makes sense to copy documentation links to a specific page and then a specific heading. Like in Laravel documentation, headings are clickable, and the URL appends with `#some_heading`. 

>  **Note** It's a small change. We can ignore it, but it's good UX as almost all docs site has this, and it's kind of become muscle memory.

**Comment by stancl** [comment]
Yeah, I'd add a clickable `#` in front of the headings. Like Laravel docs: https://laravel.com/docs/9.x/installation#laravel-and-docker

Usually the `#` is clickable and the heading isn't, but I see that in Laravel docs they are both clickable, and it's probably better UX.
